interfaces/builtin/mir_test.go

```diff
@@ -139,8 +139,8 @@ func (s *MirInterfaceSuite) TestSecCompOnClassic(c *C) {
 func (s *MirInterfaceSuite) TestUDevSpec(c *C) {
 	udevSpec := &udev.Specification{}
 	c.Assert(udevSpec.AddPermanentSlot(s.iface, s.coreSlot), IsNil)
-	c.Assert(udevSpec.Snippets(), HasLen, 1)
-	c.Assert(udevSpec.Snippets()[0], testutil.Contains, `KERNEL=="event[0-9]*", TAG+="snap_mir-server_mir"`)
+	c.Assert(udevSpec.Snippets(), HasLen, 5)
```

Comment:
Why did this change from 1 snippet to 5 snippets?

Reply:
Each snippet that tags a device is now a separate entry. Before they were one big blob.

Reply:
The test is a bit relaxed now with checking length only. I think it would be nice to have a 'Contains' check, like we do for others.